### PR TITLE
split command only if it is a string

### PR DESCRIPTION
**Pull request by julian-berbel**

### CODE REVIEW

File: lib/mumukit/isolated_environment.rb [hunk 1]
```diff
@@ -13,7 +13,8 @@ def configure!(*files)
       binds = dirnames.map { |it| "#{it}:#{it}" }
       volumes = Hash[dirnames.map { |it| [[it, {}]] }]
 
-      command = yield(*filenames).split
+      command = yield(*filenames)
+      command = command.split if String === command
```
Comment (flbulgarelli) [inline lib/mumukit/isolated_environment.rb]: Why is this necessary?

Comment (flbulgarelli) [inline lib/mumukit/isolated_environment.rb]: Please use `is_a?`, let's avoid `===` unless we really need a polymorphic behavior

Comment (julian-berbel) [reply lib/mumukit/isolated_environment.rb]: It gives us a little more control over the command to execute. For example, say we need to pass something through a command by piping it, if we allow only a string as a command, then something like `bash -c "echo something | command"` would be split to `['bash', '-c', '"echo', 'something', '|', 'command']` instead of the correct form: `['bash', '-c', 'echo something | command']`
